modules/monitoring/application/views/scripts/list/notifications.phtml

```diff
@@ -35,7 +35,7 @@ if (! $this->compact): ?>
             <td class="state-col state-<?= $stateName ?>">
                 <div class="state-label"><?= $stateLabel ?></div>
                 <div class="state-meta">
-                    <?= $this->timeAgo($notification->notification_timestamp, $this->compact) ?>
+                    <?= $this->formatDateTime($notification->notification_timestamp) ?>
```

Comment:
If we drop compact (aka timeOnly) here for compact views, does this still render in a good way on mobile screens?

Reply:
Schaut ok für mich aus
![screenshot_20180322-082237](https://user-images.githubusercontent.com/6574967/37756771-ffbadb84-2daa-11e8-89b3-3958c6a31c13.jpg)